apps/transport/lib/transport_web/live/notifications_live.ex

```diff
@@ -6,6 +6,7 @@ defmodule TransportWeb.Live.NotificationsLive do
   import TransportWeb.Gettext
   import TransportWeb.BreadCrumbs, only: [breadcrumbs: 1]
 
+  @impl true
```

Comment:
Pourquoi cela est-il désormais nécessaire/utile ?

Reply:
À ma connaissance ce n'est pas hyper utile mais ça permet de distinguer [ce qui est spécifique à LiveView](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#callbacks) et les autres méthodes https://hexdocs.pm/elixir/main/Module.html#module-impl-since-v1-5-0.

Si d'autres callbacks sont ajoutés plus tard on aura une erreur de compilation indiquant qu'on n'a pas ces méthodes.

Reply:
:+1: 

merci